pkg/webui/console/containers/top-entities-dashboard-panel/top-entities-panel.styl

```diff
@@ -44,4 +44,15 @@
     &-divided
       width: 33%
     &-small
-      width: 2.5rem
\ No newline at end of file
+      min-width: 2.5rem
+      width: 2.5rem
+
+@container panel (max-width: 360px)
+  .entity-cell-content
+    width: 35%
+    overflow: hidden
+    text-overflow: ellipsis
+  .entity-cell-last-column
+    position: absolute
+    top: 20px
+    right: 10px
```

Comment:
Please remove this and refactor the panel to not use infinite scroll but a regular scroll with a limit of 10 entities.